native-navigation/src/components/Navigation.js

```diff
@@ -104,6 +104,16 @@ class Navigation extends Component {
 		if (this.state.loading) {
 			return this.props.children[0]
 		}
+
+		const { provider, customStore } = this.props;
```

Comment:
That won't work, there's no prop called customStore

Reply:
@erikpoort Check again, I changed the "store" prop sent from index.js on the example to "customStore".
It can't be named just "store" because the compiler doesn't know what to do when it reads "store={store}" and the red screen appears.